react/players/VimeoPlayer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent } from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+
+import { VideoPlayer } from '../interfaces'
+
+const CSS_HANDLES = ['videoContainer', 'videoElement'] as const
+
+const VIMEO_REGEX = /^.*(vimeo\.com\/)((channels\/[A-z]+\/)|(groups\/[A-z]+\/videos\/))?([0-9]+)/
```

Comment:
It would be awesome to have a link to some tests for this RegEx from https://regex101.com. Similar to: https://github.com/vtex-apps/docs-ui/blob/a672461f26180471874e62b3cc36a5e20f75897d/react/Search.tsx#L29-L35. In the example, the expression is very simple, but this one would benefit more.